diff -r -c ORIG.48/configure.ac CURRENT/configure.ac
*** ORIG.48/configure.ac	Tue Mar 22 03:15:38 2016
--- CURRENT/configure.ac	Sun Apr  3 20:20:39 2016
***************
*** 1637,1644 ****
                      RAND_egd \
                      ENGINE_cleanup \
                      CRYPTO_cleanup_all_ex_data \
!                     SSL_get_shutdown \
!                     SSLv2_client_method )
  
      AC_MSG_CHECKING([for BoringSSL])
      AC_COMPILE_IFELSE([
--- 1637,1643 ----
                      RAND_egd \
                      ENGINE_cleanup \
                      CRYPTO_cleanup_all_ex_data \
!                     SSL_get_shutdown)
  
      AC_MSG_CHECKING([for BoringSSL])
      AC_COMPILE_IFELSE([
