diff -c -r ORIGs/tools/build/src/tools/msvc.jam CURRENT/tools/build/src/tools/msvc.jam
*** ORIGs/tools/build/src/tools/msvc.jam	2024-04-11 14:48:06.000000000 -0400
--- CURRENT/tools/build/src/tools/msvc.jam	2024-05-30 16:11:56.988904100 -0400
***************
*** 1137,1143 ****
          }
          else
          {
!             if [ MATCH "(14.3)" : $(version) ]
              {
                  if $(.debug-configuration)
                  {
--- 1137,1151 ----
          }
          else
          {
!             if [ MATCH "(14.4)" : $(version) ]
!             {
!                 if $(.debug-configuration)
!                 {
!                     ECHO "notice: [generate-setup-cmd] $(version) is 14.4" ;
!                 }
!                 parent = [ path.native [ path.join  $(parent) "..\\..\\..\\..\\..\\Auxiliary\\Build" ] ] ;
!             }
!             else if [ MATCH "(14.3)" : $(version) ]
              {
                  if $(.debug-configuration)
                  {
***************
*** 1316,1322 ****
              # version from the path.
              # FIXME: We currently detect both Microsoft Visual Studio 9.0 and
              # 9.0express as 9.0 here.
!             if [ MATCH "(MSVC\\\\14.3)" : $(command) ]
              {
                  version = 14.3 ;
              }
--- 1324,1334 ----
              # version from the path.
              # FIXME: We currently detect both Microsoft Visual Studio 9.0 and
              # 9.0express as 9.0 here.
!             if [ MATCH "(MSVC\\\\14.4)" : $(command) ]
!             {
!                 version = 14.4 ;
!             }
!             else if [ MATCH "(MSVC\\\\14.3)" : $(command) ]
              {
                  version = 14.3 ;
              }
***************
*** 1745,1757 ****
          # And fortunately, forward slashes do also work in native Windows.
          local vswhere = "$(root)/Microsoft Visual Studio/Installer/vswhere.exe" ;
          # The check for $(root) is to avoid a segmentation fault if not found.
!         if $(version) in 14.1 14.2 14.3 default && $(root) && [ path.exists $(vswhere) ]
          {
              local req = "-requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64" ;
              local prop = "-property installationPath" ;
              local limit ;
  
!             if $(version) = 14.3
              {
                  limit = "-version \"[17.0,18.0)\" -prerelease" ;
              }
--- 1757,1773 ----
          # And fortunately, forward slashes do also work in native Windows.
          local vswhere = "$(root)/Microsoft Visual Studio/Installer/vswhere.exe" ;
          # The check for $(root) is to avoid a segmentation fault if not found.
!         if $(version) in 14.1 14.2 14.3 14.4 default && $(root) && [ path.exists $(vswhere) ]
          {
              local req = "-requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64" ;
              local prop = "-property installationPath" ;
              local limit ;
  
!             if $(version) = 14.4
!             {
!                 limit = "-version \"[17.0,18.0)\" -prerelease" ;
!             }
!             else if $(version) = 14.3
              {
                  limit = "-version \"[17.0,18.0)\" -prerelease" ;
              }
***************
*** 2174,2180 ****
                       armv7 armv7s ;
  
  # Known toolset versions, in order of preference.
! .known-versions = 14.3 14.2 14.1 14.0 12.0 11.0 10.0 10.0express 9.0 9.0express 8.0 8.0express 7.1
      7.1toolkit 7.0 6.0 ;
  
  # Version aliases.
--- 2190,2196 ----
                       armv7 armv7s ;
  
  # Known toolset versions, in order of preference.
! .known-versions = 14.4 14.3 14.2 14.1 14.0 12.0 11.0 10.0 10.0express 9.0 9.0express 8.0 8.0express 7.1
      7.1toolkit 7.0 6.0 ;
  
  # Version aliases.
***************
*** 2226,2231 ****
--- 2242,2252 ----
      "Microsoft Visual Studio/2022/*/VC/Tools/MSVC/*/bin/Host*/*"
      ;
  .version-14.3-env = VS170COMNTOOLS ProgramFiles ProgramFiles(x86) ;
+ .version-14.4-path =
+     "../../VC/Tools/MSVC/*/bin/Host*/*"
+     "Microsoft Visual Studio/2022/*/VC/Tools/MSVC/*/bin/Host*/*"
+     ;
+ .version-14.4-env = VS170COMNTOOLS ProgramFiles ProgramFiles(x86) ;
  
  # Auto-detect all the available msvc installations on the system.
  auto-detect-toolset-versions ;
