diff -r -c ORIGs/win32/Makefile.msc CURRENT/win32/Makefile.msc
*** ORIGs/win32/Makefile.msc	Sun Apr 28 20:23:49 2013
--- CURRENT/win32/Makefile.msc	Tue Sep 24 14:51:53 2019
***************
*** 26,34 ****
  LD = link
  AR = lib
  RC = rc
! CFLAGS  = -nologo -MD -W3 -O2 -Oy- -Zi -Fd"zlib" $(LOC)
  WFLAGS  = -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE
! ASFLAGS = -coff -Zi $(LOC)
  LDFLAGS = -nologo -debug -incremental:no -opt:ref
  ARFLAGS = -nologo
  RCFLAGS = /dWIN32 /r
--- 26,34 ----
  LD = link
  AR = lib
  RC = rc
! #CFLAGS  = -nologo -MD -W3 -O2 -Oy- -Zi -Fd"zlib" $(LOC)
  WFLAGS  = -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE
! #ASFLAGS = -coff -Zi $(LOC)
  LDFLAGS = -nologo -debug -incremental:no -opt:ref
  ARFLAGS = -nologo
  RCFLAGS = /dWIN32 /r
