*** ORIGS/win32/Makefile.msc	2014-01-05 10:19:28.363940400 -0500
--- CURRENT/win32/Makefile.msc	2014-01-05 10:22:35.433219700 -0500
***************
*** 43,92 ****
       example.exe minigzip.exe example_d.exe minigzip_d.exe
  
  $(STATICLIB): $(OBJS) $(OBJA)
! 	$(AR) $(ARFLAGS) -out:$@ $(OBJS) $(OBJA)
  
  $(IMPLIB): $(SHAREDLIB)
  
  $(SHAREDLIB): $(TOP)/win32/zlib.def $(OBJS) $(OBJA) zlib1.res
! 	$(LD) $(LDFLAGS) -def:$(TOP)/win32/zlib.def -dll -implib:$(IMPLIB) \
  	  -out:$@ -base:0x5A4C0000 $(OBJS) $(OBJA) zlib1.res
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;2
  
  example.exe: example.obj $(STATICLIB)
! 	$(LD) $(LDFLAGS) example.obj $(STATICLIB)
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;1
  
  minigzip.exe: minigzip.obj $(STATICLIB)
! 	$(LD) $(LDFLAGS) minigzip.obj $(STATICLIB)
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;1
  
  example_d.exe: example.obj $(IMPLIB)
! 	$(LD) $(LDFLAGS) -out:$@ example.obj $(IMPLIB)
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;1
  
  minigzip_d.exe: minigzip.obj $(IMPLIB)
! 	$(LD) $(LDFLAGS) -out:$@ minigzip.obj $(IMPLIB)
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;1
  
  {$(TOP)}.c.obj:
! 	$(CC) -c $(WFLAGS) $(CFLAGS) $<
  
  {$(TOP)/test}.c.obj:
! 	$(CC) -c -I$(TOP) $(WFLAGS) $(CFLAGS) $<
  
  {$(TOP)/contrib/masmx64}.c.obj:
! 	$(CC) -c $(WFLAGS) $(CFLAGS) $<
  
  {$(TOP)/contrib/masmx64}.asm.obj:
! 	$(AS) -c $(ASFLAGS) $<
  
  {$(TOP)/contrib/masmx86}.asm.obj:
! 	$(AS) -c $(ASFLAGS) $<
  
  adler32.obj: $(TOP)/adler32.c $(TOP)/zlib.h $(TOP)/zconf.h
  
--- 43,92 ----
       example.exe minigzip.exe example_d.exe minigzip_d.exe
  
  $(STATICLIB): $(OBJS) $(OBJA)
! 	"$(AR)" $(ARFLAGS) -out:$@ $(OBJS) $(OBJA)
  
  $(IMPLIB): $(SHAREDLIB)
  
  $(SHAREDLIB): $(TOP)/win32/zlib.def $(OBJS) $(OBJA) zlib1.res
! 	"$(LD)" $(LDFLAGS) -def:$(TOP)/win32/zlib.def -dll -implib:$(IMPLIB) \
  	  -out:$@ -base:0x5A4C0000 $(OBJS) $(OBJA) zlib1.res
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;2
  
  example.exe: example.obj $(STATICLIB)
! 	"$(LD)" $(LDFLAGS) example.obj $(STATICLIB)
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;1
  
  minigzip.exe: minigzip.obj $(STATICLIB)
! 	"$(LD)" $(LDFLAGS) minigzip.obj $(STATICLIB)
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;1
  
  example_d.exe: example.obj $(IMPLIB)
! 	"$(LD)" $(LDFLAGS) -out:$@ example.obj $(IMPLIB)
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;1
  
  minigzip_d.exe: minigzip.obj $(IMPLIB)
! 	"$(LD)" $(LDFLAGS) -out:$@ minigzip.obj $(IMPLIB)
  	if exist $@.manifest \
  	  mt -nologo -manifest $@.manifest -outputresource:$@;1
  
  {$(TOP)}.c.obj:
! 	"$(CC)" -c $(WFLAGS) $(CFLAGS) $<
  
  {$(TOP)/test}.c.obj:
! 	"$(CC)" -c -I$(TOP) $(WFLAGS) $(CFLAGS) $<
  
  {$(TOP)/contrib/masmx64}.c.obj:
! 	"$(CC)" -c $(WFLAGS) $(CFLAGS) $<
  
  {$(TOP)/contrib/masmx64}.asm.obj:
! 	"$(AS)" -c $(ASFLAGS) $<
  
  {$(TOP)/contrib/masmx86}.asm.obj:
! 	"$(AS)" -c $(ASFLAGS) $<
  
  adler32.obj: $(TOP)/adler32.c $(TOP)/zlib.h $(TOP)/zconf.h
  
