*** ORIGS/Configurations/windows-makefile.tmpl	Thu Aug  4 10:00:44 2016
--- CURRENT/Configurations/windows-makefile.tmpl	Mon Aug 15 22:43:36 2016
***************
*** 274,280 ****
  	@rem {- output_on() unless grep { $_ eq "OPENSSL_USE_APPLINK" } @{$target{defines}}; "" -}
  	@"$(PERL)" "$(SRCDIR)\util\copy.pl" "$(SRCDIR)\include\openssl\*.h" \
  				       "$(INSTALLTOP)\include\openssl"
! 	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(BLDDIR)\include\openssl\*.h \
  				       "$(INSTALLTOP)\include\openssl"
  	@"$(PERL)" "$(SRCDIR)\util\mkdir-p.pl" "$(INSTALLTOP)\$(LIBDIR)"
  	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_LIBS) \
--- 274,280 ----
  	@rem {- output_on() unless grep { $_ eq "OPENSSL_USE_APPLINK" } @{$target{defines}}; "" -}
  	@"$(PERL)" "$(SRCDIR)\util\copy.pl" "$(SRCDIR)\include\openssl\*.h" \
  				       "$(INSTALLTOP)\include\openssl"
! 	@"$(PERL)" "$(SRCDIR)\util\copy.pl" "$(BLDDIR)\include\openssl\*.h" \
  				       "$(INSTALLTOP)\include\openssl"
  	@"$(PERL)" "$(SRCDIR)\util\mkdir-p.pl" "$(INSTALLTOP)\$(LIBDIR)"
  	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_LIBS) \
***************
*** 305,313 ****
  	@if not "$(SHLIBS)"=="" \
  	 "$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_SHLIBPDBS) \
                                          "$(INSTALLTOP)\bin"
! 	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_PROGRAMS) \
                                          "$(INSTALLTOP)\bin"
! 	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_PROGRAMPDBS) \
                                          "$(INSTALLTOP)\bin"
  	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(BIN_SCRIPTS) \
                                          "$(INSTALLTOP)\bin"
--- 305,313 ----
  	@if not "$(SHLIBS)"=="" \
  	 "$(PERL)" "$(SRCDIR)\util\copy.pl" $(INSTALL_SHLIBPDBS) \
                                          "$(INSTALLTOP)\bin"
! 	@"$(PERL)" "$(SRCDIR)\util\copy.pl" "$(INSTALL_PROGRAMS)" \
                                          "$(INSTALLTOP)\bin"
! 	@"$(PERL)" "$(SRCDIR)\util\copy.pl" "$(INSTALL_PROGRAMPDBS)" \
                                          "$(INSTALLTOP)\bin"
  	@"$(PERL)" "$(SRCDIR)\util\copy.pl" $(BIN_SCRIPTS) \
                                          "$(INSTALLTOP)\bin"
