*** VC-32-ORIG.pl	2014-05-29 22:26:19.807419200 -0400
--- VC-32.pl	2014-05-29 22:26:14.769025100 -0400
***************
*** 46,52 ****
      # 
      $base_cflags= " $mf_cflag";
      my $f = $shlib || $fips ?' /MD':' /MT';
!     $lib_cflag='/Zl' if (!$shlib);	# remove /DEFAULTLIBs from static lib
      $opt_cflags=$f.' /Ox';
      $dbg_cflags=$f.'d /Od -DDEBUG -D_DEBUG';
      $lflags="/nologo /subsystem:console /opt:ref";
--- 46,52 ----
      # 
      $base_cflags= " $mf_cflag";
      my $f = $shlib || $fips ?' /MD':' /MT';
!     $lib_cflag='/Z7' if (!$shlib);	# remove /DEFAULTLIBs from static lib
      $opt_cflags=$f.' /Ox';
      $dbg_cflags=$f.'d /Od -DDEBUG -D_DEBUG';
      $lflags="/nologo /subsystem:console /opt:ref";
***************
*** 127,133 ****
      {
      $base_cflags= " $mf_cflag";
      my $f = $shlib || $fips ?' /MD':' /MT';
!     $lib_cflag='/Zl' if (!$shlib);	# remove /DEFAULTLIBs from static lib
      $ff = "/fixed";
      $opt_cflags=$f.' /Ox /O2 /Ob2';
      $dbg_cflags=$f.'d /Od -DDEBUG -D_DEBUG';
--- 127,133 ----
      {
      $base_cflags= " $mf_cflag";
      my $f = $shlib || $fips ?' /MD':' /MT';
!     $lib_cflag='/Z7' if (!$shlib);	# remove /DEFAULTLIBs from static lib
      $ff = "/fixed";
      $opt_cflags=$f.' /Ox /O2 /Ob2';
      $dbg_cflags=$f.'d /Od -DDEBUG -D_DEBUG';
***************
*** 151,158 ****
  	}
  
  # generate symbols.pdb unconditionally
! $app_cflag.=" /Zi /Fd\$(TMP_D)/app";
! $lib_cflag.=" /Zi /Fd\$(TMP_D)/lib";
  $lflags.=" /debug";
  
  $obj='.obj';
--- 151,158 ----
  	}
  
  # generate symbols.pdb unconditionally
! $app_cflag.=" /Z7 /Fd\$(TMP_D)/app";
! $lib_cflag.=" /Z7 /Fd\$(TMP_D)/lib";
  $lflags.=" /debug";
  
  $obj='.obj';
***************
*** 198,204 ****
  		$asm='nasm -f win64 -DNEAR -Ox -g';
  		$afile='-o ';
  	} else {
! 		$asm='ml64 /c /Cp /Cx /Zi';
  		$afile='/Fo';
  	}
  } elsif ($FLAVOR =~ /WIN64I/) {
--- 198,204 ----
  		$asm='nasm -f win64 -DNEAR -Ox -g';
  		$afile='-o ';
  	} else {
! 		$asm='ml64 /c /Cp /Cx /Z7';
  		$afile='/Fo';
  	}
  } elsif ($FLAVOR =~ /WIN64I/) {
***************
*** 212,218 ****
  	$asmtype="win32n";
  	$afile='-o ';
  } else {
! 	$asm='ml /nologo /Cp /coff /c /Cx /Zi';
  	$afile='/Fo';
  	$asmtype="win32";
  }
--- 212,218 ----
  	$asmtype="win32n";
  	$afile='-o ';
  } else {
! 	$asm='ml /nologo /Cp /coff /c /Cx /Z7';
  	$afile='/Fo';
  	$asmtype="win32";
  }
