*** zlib-1.2.5.Orig\win32\Makefile.msc	2010-03-28 23:51:22.000000000 -0400
--- CURRENT\win32\Makefile.msc	2010-06-09 19:33:34.260994300 -0400
***************
*** 22,30 ****
  LD = link
  AR = lib
  RC = rc
! CFLAGS  = -nologo -MD -W3 -O2 -Oy- -Zi -Fd"zlib" $(LOC)
  WFLAGS  = -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE
! ASFLAGS = -coff -Zi $(LOC)
  LDFLAGS = -nologo -debug -incremental:no -opt:ref
  ARFLAGS = -nologo
  RCFLAGS = /dWIN32 /r
--- 22,30 ----
  LD = link
  AR = lib
  RC = rc
! CFLAGS  = -nologo -W3 -O2 -Oy- -Z7 -Fd"zlib" $(LOC)
  WFLAGS  = -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE
! ASFLAGS = -coff -Z7 $(LOC)
  LDFLAGS = -nologo -debug -incremental:no -opt:ref
  ARFLAGS = -nologo
  RCFLAGS = /dWIN32 /r
